spec/deploy_spec.rb

```diff
@@ -153,6 +152,26 @@
     end
   end
 
+  describe '#container_hostname' do
+    it 'does not provide a container hostname if no override is given' do
+      expect(test_deploy.container_hostname('foo')).to be_nil
+    end
+
+    context 'container_hostname is overridden with a string' do
+      it 'does not provide a container hostname if no override is given' do
```

Comment:
Test description here is not accurate, the same for the `it` below. Also, using something like `'server_name'` instead of `'foo'` would communicate the intent more clearly.